public/dom.js

```diff
@@ -1,37 +1,41 @@
 const locateMe = document.querySelector('.button__location');
 const alert = document.querySelector('.location__alert');
+const postcodeMe = document.querySelector('.button__postcode');
+const postcodeText = document.getElementsByClassName('text__location')[0];
 
-const geolocate = () => {
-  useLocation()
-    .then(pos => {
-      const url = `/geolocation?long=${pos.coords.longitude}&lat=${pos.coords.latitude}`;
+const geolocate = (promise) => {
+  promise()
+    .then((pos) => {
+      const url = `/geolocation?long=${pos.longitude}&lat=${pos.latitude}`;
       fetch(url)
-        .then(res => {
+        .then(() => {
           window.location.href = '/filter';
         })
-        .catch(err => {
+        .catch((err) => {
           console.log('error', err);
-        })
-    })
-    .catch(err => {
-      alert.textContent = 'Sorry, your browser is not compatible with Use My Location.';
-      locateMe.disabled = true;
+        });
     })
+    .catch((err) => {
+      if (err === 'geolocation') {
+        alert.textContent = 'Sorry, your browser is not compatible with Use My Location.';
```

Comment:
Please change to 'Sorry, your browser is not compatible with the Use My Location feature.'

Reply:
done